src/ApiClient.php

```diff
@@ -39,17 +40,32 @@ class ApiClient
      */
     private $userAgent;
 
+    private CookieJar $cookieJar;
```

Comment:
Typed properties are not supported on the minimum PHP version. Please remove the type and replace it with a PHPDoc annotation.